packages.json

```diff
@@ -1,4 +1,17 @@
 [
+  {
+    "name": "nim_tinyfiledialogs", 
+    "url": "https://github.com/Patitotective/nim_tinyfiledialogs", 
+    "method": "git", 
+    "tags": [
+      "wrapper", 
+      "bindings", 
+      "tinyfd"
```

Comment:
i think instead of generic tags like this you can add tags like:
`file dialogs`
`dialogs`


 so e.g. on nimble.directory it'll be much easier to find your package